### PR TITLE
fix: outline icons spacing and size

### PR DESCRIPTION
**Pull request by kirillkurko**
### Describe what this PR does

Few icons sizes and spacings were updated (found inconsistencies during testing)

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

DD-{number} #done

### CODE REVIEW

File: packages/shared/src/components/BookmarkFeedLayout.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default function BookmarkFeedLayout({
   const shareBookmarksButton = (style: string, text?: string) => (
     <Button
       className={style}
-      icon={<SourceIcon />}
+      icon={<SourceIcon className="icon" />}
```
Comment (sshanzel) [inline packages/shared/src/components/BookmarkFeedLayout.tsx:66]: Can you check what are the styles applied for using the icon class? I think this doesn't do anything anymore but I could be wrong.

**Review by sshanzel — APPROVED** [review]
Just some things that needs checking but looks good to me overall! ⛴️

File: packages/shared/src/components/widgets/PostUsersHighlights.tsx [hunk 2]
```diff
@@ -87,10 +87,10 @@ const UserHighlight = (props: SourceAuthorProps) => {
       {Icon && (
         <Icon
           className={classNames(
-            'absolute w-5 h-5 top-11 left-11',
+            'absolute w-5 h-5 top-10 left-10',
```
[the remark this answers is not in the document]
Comment (kirillkurko) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:90]: Yes, I checked both icons before making this change

File: packages/shared/src/components/BookmarkFeedLayout.tsx (hunk 1 above)
Comment (kirillkurko) [reply packages/shared/src/components/BookmarkFeedLayout.tsx:66]: I do not remember exactly, but it applies some font-size (doesn't really matter it this case) and some margin, so icon is not so close to the text

**Review by rebelchris — APPROVED** [review]
Looks good to me, we can evaluate the icon class later, I think only buttons use it now.

**Pull request merged**